plugin-server/src/cdp/consumers/cdp-behavioural-events.consumer.test.ts

```diff
@@ -114,9 +119,11 @@ describe('CdpBehaviouralEventsConsumer', () => {
 
         await cassandra.connect()
         const repository = processor.getBehavioralCounterRepository()!
+        const occurrenceRepository = processor.getPersonEventOccurrenceRepository()!
```

Comment:
This feels overkill. You can just do `processor['personEventOccurrenceRepository']` - private methods can still be accessed in this way which keeps your tests much cleaner